### PR TITLE
 Remove PBS_translate_mpp hook as support for mpp is ended

### PR DESCRIPTION
**Pull request by npadole20**
#### Describe Bug or Feature
PBS_translate_mpp hook was disabled and support for mpp* is ended. As part of this PR, we are removing the hook and it's config file

#### Describe Your Change
Removed PBS_translate_mpp.PY and PBS_translate_mpp.HK

#### Link to Design Doc

NA

#### Attach Test and Valgrind Logs/Output

NA

### CODE REVIEW

File: test/fw/bin/pbs_stat [hunk 1]
```diff
@@ -615,15 +615,12 @@ if __name__ == '__main__':
         hooks = []
         _hlist = server.status(HOOK)
         for hook in _hlist:
-            if (not hook['id'].startswith('PBS_translate_mpp') and
-                    not hook['id'].startswith('PBS_ibwins') and
-                    not hook['id'].startswith('PBSadd_spawn')):
```
Comment (vccardenas) [inline test/fw/bin/pbs_stat]: Is there a reason why PBS_ibwins and PBSadd_spawn were also removed in addition to PBS_translate_mpp?

**Comment by vccardenas** [comment]
It looks like some tests still refer to the translate hook which might also need to be changed:
4046_support_site_allocation_mgmt/t/t3.6.t
sp265182_support_linux_cgroup_phase1/t/t80.t
sp265182_support_linux_cgroup_phase1/t/t31.t

**Comment by lisa-altair** [comment]
Should the mpp* resources/code be removed too?  
Doing a grep on pbspro, I still see mpphost, mppwidth, mppmem, etc in:
src/lib/Libpbs/ecl_resc_def_all.c
src/lib/Libattr/master_resc_def_all.xml
src/lib/Libattr/resc_def_all.c

**Comment by bhroam** [comment]
@lisa-altair the c files are auto-generated from the XML file.  Only the XML file needs to be modified.

File: test/fw/bin/pbs_stat (hunk 1 above)
Comment (npadole20) [reply test/fw/bin/pbs_stat]: PBS_ibwins was removed in 18.x. We do not have any file PBSadd_spawn in our source code or mentioned anywhere in the documentation. So I am assuming it is deprecated too.

**Comment by npadole20** [comment]
@bhroam please review the changes made to  src/lib/Libattr/master_resc_def_all.xml

**Comment by npadole20** [comment]
@bhroam please take this PR forward.

**Pull request merged**